pandas/tests/reductions/test_reductions.py

```diff
@@ -56,23 +56,54 @@ def test_nanops(self):
         # GH#7261
         for opname in ['max', 'min']:
             for klass in [Index, Series]:
+                arg_op = 'arg' + opname if klass is Index else 'idx' + opname
 
                 obj = klass([np.nan, 2.0])
                 assert getattr(obj, opname)() == 2.0
 
                 obj = klass([np.nan])
                 assert pd.isna(getattr(obj, opname)())
+                assert pd.isna(getattr(obj, opname)(skipna=False))
 
                 obj = klass([])
                 assert pd.isna(getattr(obj, opname)())
+                assert pd.isna(getattr(obj, opname)(skipna=False))
 
                 obj = klass([pd.NaT, datetime(2011, 11, 1)])
                 # check DatetimeIndex monotonic path
                 assert getattr(obj, opname)() == datetime(2011, 11, 1)
+                assert getattr(obj, opname)(skipna=False) is pd.NaT
+
+                assert getattr(obj, arg_op)() == 1
+                result = getattr(obj, arg_op)(skipna=False)
+                if klass is Series:
+                    assert np.isnan(result)
+                else:
+                    assert result == -1
 
                 obj = klass([pd.NaT, datetime(2011, 11, 1), pd.NaT])
                 # check DatetimeIndex non-monotonic path
                 assert getattr(obj, opname)(), datetime(2011, 11, 1)
+                assert getattr(obj, opname)(skipna=False) is pd.NaT
+
+                assert getattr(obj, arg_op)() == 1
+                result = getattr(obj, arg_op)(skipna=False)
+                if klass is Series:
+                    assert np.isnan(result)
+                else:
+                    assert result == -1
+
```

Comment:
put this test on the list for splitting / parametrizing later

Reply:
agreed